plugins/test/unit/plugins/importers/yum/test_sync.py

```diff
@@ -247,18 +409,21 @@ def test_failed_parse_repomd(self, mock_metadata_files):
 
     @mock.patch.object(metadata, 'MetadataFiles', autospec=True)
     def test_success(self, mock_metadata_files):
-        mock_metadata_instane = mock_metadata_files.return_value
-        mock_metadata_instane.downloader = mock.MagicMock()
-        self.reposync.import_unknown_metadata_files = mock.MagicMock(spec_set=self.reposync.import_unknown_metadata_files)
+        mock_metadata_instance = mock_metadata_files.return_value
+        mock_metadata_instance.revision = int(time.time()) + 60 * 60 * 24
```

Comment:
normally I would mention `freezegun` here but I think this test will take less than one day to run :smiley_cat: